lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -96,6 +96,14 @@
       token: 'sometoken'
       verify_ssl: False
 
+- name: Create a new user in ManageIQ from within a native Service
```

Comment:
Sorry, "a native Service" says nothing to me.
Consider giving an example explicitly setting an ansible var, and explaining in a comment that happens automatically when running playbooks inside ManageIQ.

I'd like the actual mechanics documented.  Ideally for each affected param, near the existing explanation (module_docs_fragments/manageiq.py) of defaulting to MIQ_USERNAME env var etc.  Something like "if a `manageiq` ansible variable is in scope, is a hash, and contains `username` key, ...".  Including priority between env var and ansible var...

The reason I'm pressing for all this is transparency and debuggability.
It worries me this is all behind-the-scenes opaque magic to the playbook writer, and at some point will bite someone and they'll not have any idea what's going on and how to debug their playbook.
If we hold this to the high standard "If the implementation is hard to explain, it's a bad idea.", it will force us to make the behavior more predictable...